app/controllers/main_controller.rb

```diff
@@ -160,7 +160,9 @@ def index
     ]).order('id ASC')
 
     @current_assignment = Assignment.get_current_assignment
-    @current_ta = @current_assignment.tas.first
+    if @current_assignment!=nil
```

Comment:
Should be:  
@current_ta = @current_assignment.tas.first if @current_assignment.present?
or 
 @current_ta = @current_assignment.tas.first unless @current_assignment.nil?